release.version

```diff
@@ -0,0 +1 @@
+0.0.1
```

Comment:
So do we version the images separately from the test container? Also, while in main branch, it should have some indication that it is not release but main branch build.

Reply:
Hmmm yes, that's true...so do we want to hardcode value with `main`? Additionally, when we do release we basically change it to `0.0.1` or something like this...